src/screens/NewWallet.js

```diff
@@ -104,13 +104,14 @@ function NewWallet() {
    */
   const pinSuccess = () => {
     // Generate addresses and load data
+    LOCAL_STORE.unlock();
     wallet.generateWallet(words, '', pin, password, history);
+    // Mark this wallet as open, so that it does not appear locked after loading
+    LOCAL_STORE.open();
     // Clean pin, password and words from redux
     dispatch(updatePassword(null));
     dispatch(updatePin(null));
     dispatch(updateWords(null));
-    // Go to wallet
-    history.push('/wallet/');
```

Comment:
How are we changing screens without this now?

Reply:
The redux has a copy of `reactHistory` object and the wallet start saga will redirect the screen to the `loading_addresses` screen when the wallet is loading.
After that it will naturally go to the loaded wallet screen since the wallet is loaded, unlocked and opened.